app/components/Explorer/Assets.jsx

```diff
@@ -37,8 +37,8 @@ class Assets extends React.Component {
                 typeof accountStorage.get(`totalAssets_${chainID}`) != "object"
                     ? accountStorage.get(`totalAssets_${chainID}`)
                     : chainID && chainID === "4018d784"
-                        ? 3000
-                        : 50, // mainnet has 3000+ assets, other chains may not have that many
+                    ? 3000
```

Comment:
What formatter are you using? Those reformatting changes seem unnecessary

Reply:
I use prettier, but it does not seem to define the relevant configuration.
I restore the code at this location

Reply:
This seems to be husky, it is done automatically.

![image](https://user-images.githubusercontent.com/6350089/146629675-26b7dbbc-683d-4ba7-91a8-71c044947ec4.png)

Reply:
This code is in package.json
"precommit": "pretty-quick --staged"

Reply:
I have restored this place back.